### ISSUE TITLE
the program does not work with screen readers such as NVDA

### ISSUE THREAD

**Issue by titet11** [Anjok07/ultimatevocalremovergui#957]
Unfortunately this program does not respond with any screen reader.

I would like it to read each of the options when I press tab and be able to scroll through each option using the arrows.
